snippets/newsletter-signup.src.html

```diff
@@ -0,0 +1,53 @@
+<div class="content-block content-block--full background-alternate">
+    <div class="section">
+        <div class="container container--narrow">
+
+            <form class="members-form" data-members-form="subscribe">
+
+                <h2>Nenechte si ujít nové&nbsp;články</h2>
+                <p>
+                    Líbí se vám článek? Zadejte svůj e-mail a&nbsp;nenechte si ujít ty příští!
+                </p>
+
+                <div class="input-group spacing-bottom-small">
+                    <label for="signup-in-article-email" class="visually-hidden">
+                        E-mail
+                    </label>
+                    <input type="email" id="signup-in-article-email" class="text-field" placeholder="Zadejte svůj e-mail" size="25" data-members-email="" required="" />
```

Comment:
Neměli bychom ještě validovat vstup od uživatele na klientovi? Regex na e-mail by mohl stačit, ne?

Reply:
Za mě KISS – nechat na prohlížeči a na backendu.